### PR TITLE
feat(server-mogno): scan entitites on startup

### PR DESCRIPTION
**Pull request by eikenilsknopp**

### CODE REVIEW

File: sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/compatibility/MorphiaCompatibilityITest.java [hunk 1]
```diff
@@ -77,7 +77,8 @@ void shouldStoreGenericTypes() {
   }
 
   @Test
-  @Disabled("This test shows the default of Spring Data MongoDB - using _class instead of Morphia's className.")
+  @Disabled(
+      "This test shows the default of Spring Data MongoDB - using _class instead of Morphia's className.")
```
Comment (preethivenkatesh-sdase) [inline sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/compatibility/MorphiaCompatibilityITest.java:81]: spotless issue?

**Review by preethivenkatesh-sdase — COMMENTED** [review]
Should we remove the Morphia prefix in the **MorphiaBundleCustomConverterIT.java** filename?

File: sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundle.java [hunk 2]
```diff
@@ -211,10 +228,38 @@ public interface InitialBuilder {
...
+      return withEntities(asList(entityClasses));
+    }
+
+    /**
+     * @param entityClasses Model classes that represent entities. Using explicit classes instead of
+     *     scanning packages boosts application startup.
+     * @return a builder instance for further configuration
+     */
+    CustomConverterBuilder<C> withEntities(@NotNull List<Class<?>> entityClasses);
+  }
```
[the remark this answers is not in the document]
Comment (eikenilsknopp) [reply sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundle.java:261]: Technicaly, the `...` option would be enough, yes. Only the naming would be a bit off when using withEntities with only a single class. If thats ok for us I would remove the other options.

**Review by christopher-cudennec — COMMENTED** [review]
Forgot to mention the documentation in the `README.md`.
Please compare our [old documentation for the Morphia bundle](https://github.com/SDA-SE/sda-dropwizard-commons/tree/master/sda-commons-server-morphia#initialization).

**Review by christopher-cudennec — APPROVED** [review]

**Pull request merged**